frontend/packages/data-portal/app/routes/datasets.$id.tsx

```diff
@@ -67,6 +67,10 @@ const GET_DATASET_BY_ID = gql(`
         }
       }
     }
+    dataset_authors(distinct_on: name, where: {dataset_id: {_eq: $id}, affiliation_name: {_is_null: false}}) {
```

Comment:
the original `authors` key in the dataset query (from the metadata drawer) filtered out all authors that did have an institutional affiliation so I extracted it into its own query since it messed with the results I was looking for (all unique authors)